docs/src/www/assets/samples/WIFI_7697_wheelrobot.ino.ino

```diff
@@ -0,0 +1,247 @@
+/*
```

Comment:
This file has two .ino extensions. It might be confusing to people, so you should rename the file with only a single .ino.